onegov/election_day/formats/common.py

```diff
@@ -142,3 +142,44 @@ def load_csv(
         )
 
     return csv, error
+
+
+# Verification utils
+def line_is_relevant(line, number, district=None):
+    if district:
+        return line.sortwahlkreis == district and line.sortgeschaeft == number
+    else:
+        return line.sortgeschaeft == number
+
+
+def validate_integer(line, col, none_be_zero=True):
+    """
+    Checks line of a csv file for a valid integer.
+
+    :param line: line object from csv reader
+    :param col: attribute of line object
+    :param none_be_zero: raises ValueError if line.col is None
+    :return: integer value of line.col
+    """
+    assert hasattr(line, col), 'Check done in load_csv'
```

Comment:
What does 'Check done in load_csv' mean? Could you elaborate here?

If you need longer texts, you can also write assertions like this:

```python
assert hasattr(line, col), f"""
     {col} was not found on {line} - this should have failed in load_csv
"""

Reply:
I forgot to drop this line, since in load_csv in common.py, checks if the expected columns are there is performed already. The comment was a remainder of that.